assets/kube-state-metrics/deployment.yaml

```diff
@@ -57,7 +57,22 @@ spec:
           ^kube_pod_completion_time$,
           ^kube_pod_status_scheduled$
         image: registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.12.0
+        livenessProbe:
+          httpGet:
+            path: livez
```

Comment:
should be /livez?

Reply:
May be [omitted](https://cs.opensource.google/go/go/+/refs/tags/go1.22.5:src/net/url/url.go;l=842), I'll patch this up though for it to be consistent with other paths.

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: sample-deployment
  namespace: default
  labels:
    app: sample-app
spec:
  replicas: 3
  selector:
    matchLabels:
      app: sample-app
  template:
    metadata:
      labels:
        app: sample-app
    spec:
		...
        - containerPort: 80
        livenessProbe:
          httpGet:
            path: healthfoo
            port: 80
			...
```
```
Containers:
  sample-container:
    Container ID:   cri-o://736a33ba57ec594200107098015e4d0fd38c03d24e46e65b359bd83e43172095
    Image:          nginx:latest
    Image ID:       docker.io/library/nginx@sha256:67682bda769fae1ccf5183192b8daf37b64cae99c6c3302650f6f8bf5f0f95df
    Port:           80/TCP
    Host Port:      0/TCP
    State:          Running
      Started:      Wed, 03 Jul 2024 23:08:25 +0530
    Ready:          False
    Restart Count:  0
    Liveness:       http-get http://:80/healthfoo delay=15s timeout=2s period=5s #success=1 #failure=3
    Readiness:      http-get http://:80/readyz delay=5s timeout=2s period=5s #success=1 #failure=3
```